src/Config/Validator/Deploy/DeprecatedVariables.php

```diff
@@ -69,7 +85,7 @@ public function validate(): Validator\ResultInterface
         }
 
         if ($this->environment->getEnv(DeployInterface::VAR_STATIC_CONTENT_THREADS)
-            || isset($variables[DeployInterface::VAR_STATIC_CONTENT_THREADS])
```

Comment:
It's not correct to change `$variables` to `$config` here, as `$config` is a merged config and VAR_STATIC_CONTENT_THREADS already removed from merged config
@see src/Config/Stage/Deploy/EnvironmentConfig.php:104 .
The same for VAR_DO_DEPLOY_STATIC_CONTENT variable
@see src/Config/Stage/Deploy/EnvironmentConfig.php:86